Unity/Assets/Scripts/HyBidAdView.cs

```diff
@@ -8,52 +8,30 @@ public abstract class HyBidAdView : MonoBehaviour
 {
     public string appToken;
 	public string placement;
-	protected string adID = "";
```

Comment:
why did you remove listeners?

Reply:
Because once ad is loaded ,its callbacks is inside native code and send response to virtual methods inside unity

Reply:
but there are fields in Unity bridges, for each add type, for that and those attributes were taken from there. As I remember you do not have to hardcode it here. Please re-check that moment.

Reply:
I would double check that moment. Those changes might break one of the platform sides functionality